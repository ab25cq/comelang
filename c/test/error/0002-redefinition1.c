/*
PATTERN: redefinition
PATTERN: c:7:
*/

int x = 0;
int x = 1;

