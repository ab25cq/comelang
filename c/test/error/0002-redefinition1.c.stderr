x already initialized at test/error/0002-redefinition1.c:7:1
int x = 1;
^
